PowerFGT/Public/cmdb/firewall/vipgroup.ps1

```diff
@@ -0,0 +1,592 @@
+#
+# Copyright 2019, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+# Copyright 2019, Benjamin Perrier <ben dot perrier at outlook dot com>
+#
+# Contribution by Brett Pound <brett underscore pound at hotmail dot com>, March 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+function Add-FGTFirewallVipGroup {
+
+    <#
+        .SYNOPSIS
+        Add a FortiGate VIP Group
+
+        .DESCRIPTION
+        Add a FortiGate VIP Group
+
+        .EXAMPLE
+        Add-FGTFirewallVipGroup -name MyVipGroup -member MyVip1 -interface wan1
+
+        Add VIP Group with member MyVip1, associated to interface wan1
+
+        .EXAMPLE
+        Add-FGTFirewallVipGroup -name MyVipGroup -member MyVip1, MyVip2 -interface wan1
+
+        Add VIP Group with members MyVip1 and MyVip2, associated to interface wan1
+
+        .EXAMPLE
+        Add-FGTFirewallVipGroup -name MyVipGroup -member MyVip1 -comment "My VIP Group" -interface wan1
+
+        Add VIP Group with member MyVip1 and a comment, associated to interface wan1
+    #>
+
+    Param(
+        [Parameter (Mandatory = $true)]
+        [string]$name,
+        [Parameter (Mandatory = $true)]
+        [string[]]$member,
+        [Parameter (Mandatory = $true)]
```

Comment:
interface is really needed ?

Reply:
can you check the API docs to confirm it's shown as mandatory there? I am pretty sure that you can't define it without a bound interface via  GUI/CLI... just checked and confirmed that Interface is required: 
config firewall vipgrp
    edit "Brett_Test"
(Brett_Test) # next
Attribute 'interface' MUST be set.
Command fail. Return code 1

Reply:
it is no specified on API :-/ 
after, we can use the same idea like for VIP cmdlet (and also GUI), set any by default  https://github.com/FortiPower/PowerFGT/blob/master/PowerFGT/Public/cmdb/firewall/vip.ps1#L49